i18n/en.json

```diff
@@ -138,11 +136,12 @@
   "I18N_TOPNAV_CONTACT_US": "Contact Us",
   "I18N_TOPNAV_LIBRARY": "Library",
   "I18N_TOPNAV_FORUM": "Forum",
+  "I18N_ACTION_BROWSE_EXPLORATIONS": "Browse our Explorations",
+  "I18N_ACTION_CREATE_EXPLORATION": "Create an Exploration",
```

Comment:
Let's keep the language from the splash page here: "Browse our explorations", "Create your own exploration"